integration-tests/tests/link-to-new-workspace.js

```diff
@@ -1,5 +1,5 @@
 const { checkbox, click, clickable, clickTableCell, fillIn, waitForNoSpinners } = require('../utils/integration-utils')
-const { enableDataCatalog, testWorkspaceName } = require('../utils/integration-helpers')
+const { checkBucketAccess, enableDataCatalog, testWorkspaceName } = require('../utils/integration-helpers')
 const { withUserToken } = require('../utils/terra-sa-utils')
 
 
```

Comment:
Could you replace lines 12-13 with the following?

```javascript
await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: 'Link to a workspace' })) })
```

I cannot make the direct comment/suggestion on those lines in gh since they were not changed in the diff.